Specifications/FeelbotTalking.feature

```diff
@@ -0,0 +1,6 @@
+Scenario : FeelBotTalking1
+One scenario of FeelBot talking
+
+Given : that FeelBot is able to aknowledge words and analyze it
```

Comment:
Food for thoughts : if the "Given" could be similar to a scenario description (so that we can "logically chain" the scenarios)

Eg 
Scenario : Blabla2
This is what happens in Blabla2

Given : what happens in Blabla1